src/main/java/net/datafaker/DateAndTime.java

```diff
@@ -250,14 +269,13 @@ public String birthday(String pattern) {
      * @return a random birthday between {@code minAge} and {@code maxAge} years ago from now.
      * @throws IllegalArgumentException if the {@code maxAge} is lower than {@code minAge}.
      */
-    public Date birthday(int minAge, int maxAge) {
-        int currentYear = Calendar.getInstance().get(Calendar.YEAR);
-        int currentMonth = Calendar.getInstance().get(Calendar.MONTH);
-        int currentDay = Calendar.getInstance().get(Calendar.DAY_OF_MONTH);
-        Calendar from = new GregorianCalendar(currentYear - maxAge, currentMonth, currentDay);
-        Calendar to = new GregorianCalendar(currentYear - minAge, currentMonth, currentDay);
-
-        return between(from.getTime(), to.getTime());
+    public LocalDate birthday(int minAge, int maxAge) {
```

Comment:
Same here. I'd prefer this to be in a separate provider, and keep this one as it, or maybe make it Deprecated or so